playbooks/ansible-test-cloud-integration-base/run.yaml

```diff
@@ -0,0 +1,101 @@
+        _test_options: "--continue-on-error --diff --requirements"
+
+    - name: Enable --no-temp-workdir for test_options
+      set_fact:
+        _test_options: "{{ _test_options }} --no-temp-workdir"
+      when: zuul.projects['github.com/ansible/ansible'].checkout not in ["stable-2.6", "stable-2.7"]
+
+    - name: Setup base target for ansible-test
+      set_fact:
+        _target: "vcenter_folder"
+#
+#    - name: Enable netconf_.* target for ansible-test
+#      set_fact:
+#        _target: "{{ _target }} netconf_.*"
+#      when: ansible_test_network_integration in ["iosxr", "junos"]
+#
+#    - name: Setup testing for collections
+#      block:
+#        - name: Setup download-artifact-fork role
+#          include_role:
+#            name: download-artifact-fork
+#          vars:
+#            download_artifact_directory: ~/downloads
+#            download_artifact_type:
+#              - ansible_collection
+#
+#        - name: Install jq
+#          become: true
+#          package:
+#            name: jq
+#            state: present
+#
+#        - name: Install ara into virtuelenv
+#          shell: ~/venv/bin/pip install yq
+#
+#        - name: Install require-project collection using ansible-galaxy
+#          args:
+#            chdir: "{{ ansible_user_dir }}/downloads"
+#            executable: /bin/bash
+#          shell: "source ~/venv/bin/activate; {{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/bin/ansible-galaxy collection install -p ~/.ansible/collection {{ item.url | basename }}"
+#          with_items: "{{ zuul.artifacts }}"
+#          when: "'metadata' in item and 'type' in item.metadata and (item.metadata.type == 'ansible_collection')"
+#
+#        - name: Get collection namespace
+#          args:
+#            chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+#            executable: /bin/bash
+#          shell: cat galaxy.yml | ~/venv/bin/yq -y .namespace | tail -n +1 | head -1
+#          register: _collection_namespace
+#
+#        - name: Get collection name
+#          args:
+#            chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+#            executable: /bin/bash
+#          shell: cat galaxy.yml | ~/venv/bin/yq -y .name | tail -n +1 | head -1
+#          register: _collection_name
+#
+#        - name: Setup base target for ansible-test
+#          set_fact:
+#            _target: "{{ ansible_test_network_integration }}_.*"
+#
+#        - name: Setup location of project
+#          set_fact:
+#            _test_location: "~/.ansible/collection/ansible_collections/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}"
+#
+#        - name: network-integration.cfg
+#          shell: "cp {{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/test/integration/network-integration.cfg {{ _test_location }}/tests/integration/network-integration.cfg"
+#
+#      when: ansible_test_collections | default(False)
+
+    - copy:
```

Comment:
Lets switch this to template, so we can change variables to be dynamic.

Reply:
It's an ini file, we can use the `ini_file` module instead.